turbinia/workers/binary_extractor.py

```diff
@@ -93,22 +94,56 @@ def run(self, evidence, result):
     self.json_path = os.path.join(self.binary_extraction_dir, 'hashes.json')
 
     cmd = [
-        'image_export.py', '--partitions', 'all', '--no_vss', '--signatures',
-        'elf,exe_mz', '--logfile', image_export_log
+        'image_export.py', '--partitions', 'all', '--no_vss', '--logfile',
+        image_export_log
     ]
+
+    if evidence.config and evidence.config.get('binary_extraction_path'):
+      artifact_dir = os.path.join(self.tmp_dir, 'artifacts')
+      artifact_file = os.path.join(artifact_dir, 'artifacts.yaml')
+      os.mkdir(artifact_dir)
```

Comment:
Can we run into a situation where the artifacts dir already exists and this will fail? If so, may make sense to check for the path first

Reply:
This shouldn't happen as the `self.tmp_dir` is created just for this task and should be empty when the task code starts.